src/components/Header/Header.tsx

```diff
@@ -9,94 +10,96 @@ import CartIcon from '@assets/icons/shopping_cart_01.svg?react';
 import LightThemeIcon from '@assets/icons/sun.svg?react';
 import SingUpIcon from '@assets/icons/user_add.svg?react';
 
+import { ROOT_URL } from '@/App';
 import { useCartContext } from '@/hooks/useCartContext';
 import { useThemeContext } from '@/hooks/useThemeContext';
 import { PageName } from '@/interfaces/Pages';
 import { PageTheme } from '@/interfaces/Themes';
+import { BurgerMenu } from '@/ui/BurgerMenu/BurgerMenu';
 
 import styles from './header.module.css';
 
-interface HeaderProps {
-    onChangePage: (page: PageName) => void;
-    activePage: PageName;
-}
-
-function Header({ onChangePage, activePage }: HeaderProps) {
+function Header() {
     const { cartData } = useCartContext();
     const { theme, handleChangeTheme } = useThemeContext();
+    const [isBurgerMenuOpen, setIsBurgerMenuOpen] = useState(false);
```

Comment:
write a useToggle hoo wich will implement togle logic
if true => false
if false => true
and use it for yr burger menu